src/test/java/software/amazon/documentdb/jdbc/DocumentDbDatabaseMetaDataTest.java

```diff
@@ -186,6 +190,26 @@ void testGetTables() throws SQLException {
         }
     }
 
+    /**
+     * Test getTables returns empty ResultSet.
+     */
+    @Test
+    @DisplayName("Test getTables returns empty ResultSet.")
+    void testGetMetadataTablesEmpty() throws SQLException {
+        // Catalog pattern, schema pattern, table name pattern, table types
+        final String[][] tests = new String [][]{
+                {null, "", COLLECTION_BASIC, "TABLE"},
+                {" ", "", COLLECTION_BASIC, "TABLE"},
+                {"non-existing catalog", DATABASE, COLLECTION_BASIC, "non-existing table"},
+                {" ", "", COLLECTION_BASIC},
+        };
+        for (String[] test : tests) {
+            final String[] tableTypes = test.length == 4 ? new String[]{test[3]} : null;
```

Comment:
Seems test[0], test[1] and test[2] are not useful here.

Reply:
good catch! I forgot to replace `null, "", COLLECTION_BASIC` `with test[0], test[1] and test[2]`